README.md

```diff
@@ -17,6 +17,21 @@ docker build --tag provideq-toolbox-backend --build-arg GAMS_LICENSE=$(base64 -w
 docker run --publish 8080:5000 provideq-toolbox-backend
 ```
 
+## Releasing a new version
+1. Create a release branch from develop: `git checkout -b release/x.y.z`.
+2. Bump the version number in the `build.gradle` file to the new version number and commit it to the release branch.
+3. Push to GitHub and create a pull request to merge the release branch into `main`.
+4. Make sure to test your new version!
```

Comment:
I'd put this comment above pushing anything to GitHub

Reply:
At some point we should write some contribution guidelines but these steps are specifically for releasing a new version (i.e., you don't need to bump the version every time you contribute). These docs however are meant to remind us of the numerous steps involved in releasing with their specific details (such as the target branch of the PR, the tagging and merging the changes back into `develop` afterwards).

Reply:
I understand, but I'd still make the point that you should test your new version before pushing the branch to GitHub and creating a PR :)

Reply:
Yes but it's particularly important when releasing a new version! I'll add a contribution guide though so that we can agreen on something :sweat_smile: